### DISCUSSION TITLE
Can authme be integrated? You can use the website to register directly, rather than users can only register in the game.

### DISCUSSION THREAD

**Discussion by 56025192** [NamelessMC/Nameless#2469] [Ideas]
### Describe the feature

Can authme be integrated? You can use the website to register directly, rather than users can only register in the game.

**Comment by samerton** [comment]
_Personally_ I'd rather that Nameless does not update any external databases (such as inserting new users into Authme), keeping it only one-way from AuthMe -> Nameless

An external module could do a sync the other way though
